astronomer/airflow/version_check/update_checks.py

```diff
@@ -309,6 +309,7 @@ def parse_version(rel):
                 url=release.get('url'),
                 description=release.get('description'),
                 end_of_support=end_of_support,
+                yanked=release.get('yanked', False),
```

Comment:
Don't we want to look at the current version to see if it's yanked, not if the version being upgraded to is yanked?

On that note, probably should ignore releases that are yanked :) don't want to suggest that as an available upgrade.

Reply:
If the current version is yanked it should show a warning. I have added yanked as part of yield in current version.

Reply:
@jedcunningham when you say we should ignore releases that are yanked, do you mean we should not show warning?

Reply:
> when you say we should ignore releases that are yanked, do you mean we should not show warning?

I mean that when looking for possible upgrades, if it's yanked, don't show it to the user. e.g. I'm on 10.6, 10.7 is out but yanked, don't show me 10.7 as an available upgrade.

> If the current version is yanked it should show a warning. I have added yanked as part of yield in current version.

Looks like I commented in the wrong spot for this (_process_update_json vs available_update), sorry. But still we are [returning the yanked flag from the newer version, not the current one](https://github.com/astronomer/astronomer-airflow-version-check/pull/74/files#diff-f00c5dd956ae403a4b5cccf02dae5b790dd874aa87fdfb74a129817bd07256daR433).

We also shouldn't be dependent on there being a new version available to show that the current version is yanked.

Reply:
Thats correct. I have done the change here: https://github.com/astronomer/astronomer-airflow-version-check/pull/74/commits/a83604a7b2a1e5dfacce9a0d008c7e1ad6c13deb